File: WotBlitzStatician.Model/Dto/AccountInfoDto.cs
using System;
using System.Collections.Generic;

namespace WotBlitzStatician.Model.Dto
{

	public class AccountInfoDto
    {
		public long AccountId { get; set; }

		public string NickName { get; set; }

		public DateTime AccountCreatedAt { get; set; }

		public DateTime LastBattleTime { get; set; }

		public PlayerClanInfoDto PlayerClanInfo { get; set; }

		public PlayerStatDto PlayerStatistics { get; set; }

	    public List<AccountMasteryInfoDto> AccountMasteryInfo { get; set; }
	}
}

File: WotBlitzStatician/client-src/app/model/account-info-dto.ts
import { PlayerClanInfo } from "./player-clan-info";
import { PlayerStatistics } from "./player-statistics";
import { AccountMasteryInfo } from "./account-mastery-info";

export interface AccountInfoDto {
    accountId: number;
    nickName: string;
    lastBattleTime: Date;
    accountCreatedAt: Date;
    playerClanInfo: PlayerClanInfo;
    playerStatistics: PlayerStatistics;
    accountMasteryInfo: AccountMasteryInfo[];
  }
  